content/100-getting-started/02-setup-prisma/100-start-from-scratch/100-connect-your-database.mdx

```diff
@@ -72,8 +72,8 @@ Note that the default schema created by `prisma init` uses PostgreSQL, so you fi
 
 ```prisma file=prisma/schema.prisma
 datasource db {
-✎  provider = "mysql"
-  url      = env("DATABASE_URL")
+  ✎  provider = "mysql"
```

Comment:
This is a custom "annotation" we use to highlight the line that should be highlighted in the rendering. It is _not_ part of the default Prisma schema syntax of course, so it makes sense that the prettier plugin does not know how to handle that.

Not super sure what to do here. Is there way to tell the plugin to ignore specific characters and treat them as invisible? That might be easiest (although we might also need to do that via our formatter/datamodel code then...)

Reply:
@janpio yeah, I just stumbled to this annotation too and was checking source code, there are already [highlighted](https://www.prisma.io/docs/about/style-guide/mdx-examples#code-block-with-highlighted-code) code support, why don't we change to that first? in another PR ofc this one already gigantic.

Reply:
I think both options are used in different cases. I generally prefer to have the highlight inline in the code example to be honest and dislike the `highlight` etc as I have to count lines.

But it might possibly be useful to indeed change that completely to make usage of this plugin possible.

Do you see another option on how to ignore that character somehow?

Reply:
> to be honest and dislike the highlight etc as I have to count lines.

yeah, I kinda experience it myself now https://github.com/prisma/docs/pull/1986

> Do you see another option on how to ignore that character somehow?

As I've checked there are different usages of code blocks with `js`, `ts`, `schema` files. I don't see any general solution to avoid the parsing issues except `highlight` annotation.

Reply:
> I don't see any general solution to avoid the parsing issues except highlight annotation.

😭